### DISCUSSION TITLE
Backup Solutions

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#5664] [Unsupported Versions]
We use Veeam as a backup service for the rest of the infrastructure we have and has the ability to backup Linux boxes. Was looking to back up the manager and search nodes in the rare event that they all die (all physical tin) and unable to pull even the backups that are mentioned in this https://docs.securityonion.net/en/2.3/backups.html I noticed while going through installing the agent that there's a version lock that is stopping me from installing the agent. while I understand why this is there if I bypass it to install the agent will it break SO? as I am unfamiliar with CentOS Or if anyone has any other suggestions to having them fully backed up would be helpful.

**Comment by dougburks** [comment]
`/nsm/backup/` contains sensitive information and therefore the permissions are set to only allow access by `root`.

What exactly is the version lock that you're seeing?

In general, we don't recommend or support bypassing version locks.

**Comment by Hammer-Bear** [comment]
so I've downloaded the packages and when running `yum install` it gives the below.

`yum install veeam
Loaded plugins: versionlock
Excluding 5 updates due to versionlock (use "yum versionlock status" to show them)
Nothing to do`

**Reply by dougburks** [reply]
Per the message, have you tried running `yum versionlock status`?

**Reply by dougburks** [reply]
@IzacFrank Please avoid asking the same question in multiple discussions.
